foundations/likvid-prod/meshstack/terragrunt.hcl

```diff
@@ -0,0 +1,72 @@
+terraform {
+  source = "${get_repo_root()}//kit/foundation/meshstack"
+
+  # todo: we should source that from a foundation.yml somehow?
+  extra_arguments "profile" {
+    commands = [
+      "init",
+      "apply",
+      "plan",
+      "import",
+      "state"
+    ]
+
+    env_vars = get_env("CI", "false") == "true" ? {} : {
+      # activate a local CLI profile when not in CI
+      AWS_PROFILE = "likvid-prod"
+    }
+  }
+}
+
+#TODO: maybe move this to an own bucket or use a different prefix
+remote_state {
```

Comment:
d: todo add comment

we store foundation module terraform state with the aws platform state

d: replaces gcs with s3 bucket config